### ISSUE TITLE
[Faceted Filters] Text features insufficient color contrast

### ISSUE THREAD

**Issue by svinkle** [Shopify/dawn#116]
## Issue summary

The Product Count text features insufficient color contrast. The color is:

- Foreground: `#878886`
- Background: `#ffffff`
- Contrast ratio: `3.56:1`

<details>
  <summary>Screenshots</summary>
<img width="412" alt="Screen shot of faceted filters section. Product count text, Showing 74 of 74 products, is highlighted." src="https://user-images.githubusercontent.com/1392632/124608554-d3568980-de3c-11eb-9c49-b361a407e568.png">

</details>

#### HTML

```html
<p class="collection-product-count light">Showing 74 of 74 products</p>
```

## Steps to reproduce

1. Go to a collection page with Dawn theme with any supported browser.
2. Check the [color contrast of the Product Count text](https://contrast-ratio.com/#%23878886-on-white).

### Expected

- Color contrast to meet AA requirements.

### Actual

- Color contrast does not meet AA requirements.

## Recommendation

Remove the `light` class in order to increase the color contrast.

#### HTML

```html
<h2>
  <span class="collection-product-count" role="status">
    Showing 74 of 74 products
  </span>
</h2>
```

<!-- #### Diff

```diff

``` -->

## Specifications

- **Component:** Faceted Filters
- **WCAG Principal:** Perceivable
- **WCAG SC:** [1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
- **Severity:** Medium
- **Effort:** Low
- **Related:** (#115, #111)

**Comment by gregjotau** [comment]
![Screenshot 2021-07-30 at 19 33 05](https://user-images.githubusercontent.com/13047497/127690612-a1861fc4-99f1-49ea-8037-d55bb3ccd2ec.png)

Insufficient contrast is a problem more places as well. Ran lighthouse for famme.no

**Comment by svinkle** [comment]
@gregjotau These may stem from customized theme settings. My concern is theme default colors and settings.

**Comment by gregjotau** [comment]
I do not think we have changed any defaults actually, but I might be wrong :)

**Comment by svinkle** [comment]
Fixed (not sure when.)
